test/unit/itext.js

```diff
@@ -1,735 +1,770 @@
 (function() {
   var canvas = this.canvas = new fabric.Canvas();
 
-  QUnit.module('fabric.IText', {
-    afterEach: function() {
+  QUnit.module('fabric.IText', function(hooks) {
+    hooks.afterEach(function() {
```

Comment:
i m not expert with QUnit, can you explain me this change?

Reply:
I don't remember perfectly but I split up those tests in three blocks: independent of retina, retinaEnabled false and retinaEnabled true. I needed a way to specify the afterEach block for all tests and then the beforeEach calls for the nested modules. I found it this way in the QUnit docs but perhaps there are other ways. I think the reason I changed the line you commented on is to make the hooks calls the same throughout the file. Not quite sure anymore, sorry!